docs/library/uctypes.rst

```diff
@@ -11,7 +11,7 @@ module is to define data structure layout with about the same power as the
 C language allows, and the access it using familiar dot-syntax to reference
 sub-fields.
 
-.. seealso::
+.. see also::
```

Comment:
I think that this should be left as seealso wihch is a spinx directive:
http://www.sphinx-doc.org/en/stable/markup/para.html#directive-seealso